docs/release-notes/8.5.asciidoc

```diff
@@ -0,0 +1,78 @@
+* Adds the option to bulk edit rule schedules to the bulk actions menu in the Rules table ({pull}140166[#140166]).
+* Adds the option to bulk edit rule actions to the bulk actions menu in the Rules table ({pull}138900[#138900]).
+* Adds an alert count card to the User, Host, and Network detail pages. The card shows alerts per rule and can be filtered by alert status ({pull}140150[#140150]).
+* Enables the Alerts related by process ancestry section by default. It appears in the Alert details flyout if you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription]. Also allows users with a https://www.elastic.co/pricing[Platinum or Enterprise subscription] to examine alerts associated with events ({pull}140006[#140006]).
+* Enables the Alerts related by session ID section by default. It appears in the Alert details flyout if you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
+* Renames the Elastic Endpoint and Cloud Security integration to the Elastic Defend integration ({pull}139517[#139517]).
+* Adds preconfigured use-cases to the setup wizard for the Elastic Defend integration (formerly known as Endpoint and Cloud Security), each with different default settings ({pull}139230[#139230]).
+* Updates the UI for the rule details flyout's *Exceptions* tab ({pull}138770[138770]).
+* Enables the Osquery Response Action and adds an *Osquery Results* tab to the Alert details flyout. You can use the Osquery Response Action to immediately query hosts that generate alerts ({pull}133279[#133279]).
+* Enables rule exceptions to reference value lists, regardless of rule type. One caveat is that text type value lists still do not work for EQL and threshold rules ({pull}133254[#133254]).
+* Introduces the new alert renderer, which concisely displays a detailed summary of the `kibana.alert.reason` field. It appears in Timeline, throughout the Alerts page, and on the alert details flyout ({pull}140825[#140825]).
+
+[discrete]
+[[bug-fixes-8.5.0]]
+==== Bug fixes and enhancements
+* Fixes a bug that sometimes caused event correlation rule (EQL) errors whenever rule queries contain regular expressions using wildcard fields and predefined character classes (for example, `\w`, `\s`, `\d`) (https://github.com/elastic/elasticsearch/pull/90064[#90064]).
+* Adds the `has_guide` tag to all prebuilt rules with investigation guides. Users can filter the Rules table by this tag to quickly find prebuilt rules with investigation guides (https://github.com/elastic/detection-rules/pull/2297[2297]).
+* Informs you when the event analyzer's current time range is too narrow to include event data ({pull}140831[#140831]).
+* Lets you inspect bar charts and data grids, as with other data visualizations ({pull}140810[#140810]).
+* Makes the Indicators table sortable by any column ({pull}140582[#140582]).
+* Provides the ability to add fields to Indicators table ({pull}138882[#138882]).
+* Updates the rule preview UI ({pull}140221[#140221]).
```

Comment:
This should probably be in "Features," even if it's technically an enhancement; it's Kind Of A Big Deal. Here's a juiced-up version:

```suggestion
* Updates the rule preview UI to be available at any step of creating or editing a detection rule, and rule previews are now possible for Elastic prebuilt rules. Rule previews also now include exceptions and field overrides ({pull}140221[#140221]).
```